### PR TITLE
feat: vec-92 use pre-embedded quote data set

### PR DESCRIPTION
**Pull request by dwelch-spike**

### CODE REVIEW

File: quote-semantic-search/quote-search/indexer.py [hunk 1]
```diff
@@ -97,14 +99,14 @@ def index_data():
 
 def index_quote(id_quote):
     id, quote = id_quote
-    quote, author, catagory = quote
+    quote, author, catagory, embedding = quote
+    embedding = ast.literal_eval(embedding)
```
Comment (jdogmcsteezy) [inline quote-semantic-search/quote-search/indexer.py:103]: need a comment to say what this is doing

Comment (jdogmcsteezy) [reply quote-semantic-search/quote-search/indexer.py:103]: We need this to be really easy code to understand

File: quote-semantic-search/scripts/pre-embed.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import csv
+import itertools
+import os
+import logging
+import tarfile
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.CRITICAL)
+
+# The image or text encoding model.
```
Comment (jdogmcsteezy) [inline quote-semantic-search/scripts/pre-embed.py:15]:
```suggestion
# The text encoding model.
```

File: quote-semantic-search/scripts/requirements.txt
Comment (jdogmcsteezy) [inline quote-semantic-search/scripts/requirements.txt:1]: This file is not needed. It has dependencies also quote-search/requirements.txt and they should remain in sync.

File: quote-semantic-search/quote-search/indexer.py [hunk 3]
```diff
@@ -97,14 +99,14 @@ def index_data():
 
 def index_quote(id_quote):
     id, quote = id_quote
-    quote, author, catagory = quote
+    quote, author, catagory, embedding = quote
```
Comment (jdogmcsteezy) [inline quote-semantic-search/quote-search/indexer.py:102]: add comment to explain that quote is pre-embedded. Good to mention it incase they did not read the README

File: quote-semantic-search/scripts/pre-embed.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+model = SentenceTransformer("sentence-transformers/all-MiniLM-L6-v2")
+MODEL_DIM = 384
```
Comment (jdogmcsteezy) [inline quote-semantic-search/scripts/pre-embed.py:17]: Anyway to pull this info from indexer.py? These should also stay in sync

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]
small changes

File: quote-semantic-search/scripts/requirements.txt
Comment (dwelch-spike) [reply quote-semantic-search/scripts/requirements.txt:1]: Deleted it.

File: quote-semantic-search/quote-search/indexer.py (hunk 3 above)
Comment (dwelch-spike) [reply quote-semantic-search/quote-search/indexer.py:105]: Left a detailed comment.

**Review by jdogmcsteezy — APPROVED** [review]
bigger comment than I was expecting :) feel free to shorten it if you wish haha

**Pull request merged**